### PR TITLE
Use system glew library if available

### PR DESCRIPTION
**Pull request by guoyunhe**
This will make build on Linux a bit faster and the final bundle smaller.
Good for Linux distribution to package the software.

### CODE REVIEW

File: build_cmake/FrameworkOpenGl/CMakeLists.txt [hunk 1]
```diff
@@ -16,13 +16,19 @@ endif()
...
 set(USE_GLEW OFF)
 if(NOT USE_GLES AND NOT APPLE)
-	if (NOT TARGET glew_s)
-		add_subdirectory(
-			${CMAKE_CURRENT_SOURCE_DIR}/../../../Dependencies/glew-2.0.0
-			${CMAKE_CURRENT_BINARY_DIR}/glew-2.0.0
-		)
+	find_package(GLEW)
+	if(GLEW_FOUND)
+		list(APPEND PROJECT_LIBS ${GLEW_LIBRARIES})
```
Comment (Zer0xFF) [inline build_cmake/FrameworkOpenGl/CMakeLists.txt]:
```suggestion
		list(APPEND PROJECT_LIBS GLEW::GLEW})
```

File: build_cmake/FrameworkOpenGl/CMakeLists.txt [hunk 2]
```diff
@@ -16,13 +16,19 @@ endif()
...
+		include_directories(${GLEW_INCLUDE_DIRS})
```
Comment (Zer0xFF) [inline build_cmake/FrameworkOpenGl/CMakeLists.txt]:
```suggestion
```

**Comment by Zer0xFF** [comment]
increase cmake version to 3.1 for the suggested changes to be valid.

**Comment by jpd002** [comment]
Thanks!

**Comment by Tatsh** [comment]
There's a syntax error on line 21. An extra `}`, but even if I remove that, it doesn't appear `GLEW::GLEW` means anything on my system (Gentoo).

When I replace it with `${GLEW_LIBRARIES}` as it was before, it builds fine.

**Comment by Zer0xFF** [comment]
the `GLEW::GLEW` is a "built-in" cmake alias, if its not working, you could be using an older version of cmake?

**Comment by guoyunhe** [comment]
I get the same result: `GLEW::GLEW` not work, but `GLEW_LIBRARIES` work. The cmake version is 3.15.5.

**Comment by guoyunhe** [comment]
The bundled cmake modules are too out-dated:

https://github.com/jpd002/Play-Dependencies/blob/master/cmake-modules/FindGLEW.cmake

Deleting them works.

**Comment by guoyunhe** [comment]
@Tatsh can you try https://github.com/jpd002/Play-Dependencies/pull/25

**Comment by guoyunhe** [comment]
@Tatsh if you run `git submodule update --init --recursive`, GLEW should work.

**Comment by Tatsh** [comment]
That works. Thanks

**Pull request merged**